units/first-boot.target

```diff
@@ -0,0 +1,15 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=First boot completed
+Documentation=man:systemd.special(7)
+After=sysinit.target
```

Comment:
I got a dependency cycle without it because `first-boot.target` is currently pulled in by `sysinit.target` ... But with the changes to the dependency structure, I guess this will no longer be necessary.